README

```diff
@@ -88,7 +88,9 @@ Advanced options:
   code or signal+128.  This also makes timeout to wait until the controlled
   process is terminated.  Without this option, the script returns zero.
 
-* `--memlimit-rss`, `-s` - monitor RSS (resident set size) memory limit
+* `--memlimit-rss`, `-s` - monitor RSS (resident set size) memory limit.
+Memory limits may be specified as KB (default), MB, or GB.
```

Comment:
I like that you documented that the default is Kilobytes.  These days, however, it's not clear if that means 1000 bytes or 1024 bytes; can you please clarify?